cardigan/stories/components/PageHeader/PageHeader.stories.tsx

```diff
@@ -167,7 +167,7 @@ const ExhibitionContentTypeInfo = () => (
 );
 
 const BookContentTypeInfo = () => (
-  <p className={`no-margin ${font('intb', 3)}`}>
+  <p className={`${font('intb', 3)}`} style={{ marginBottom: 0 }}>
```

Comment:
np; Could it be?

` <p className={font('intb', 3)} style={{ marginBottom: 0 }}>`